app/src/js/components/Collections/overview.js

```diff
@@ -13,30 +14,30 @@ import {
   listGranules,
   searchGranules,
 } from '../../actions';
-import { get } from 'object-path';
 import {
   collectionName as collectionLabelForId,
-  tally,
-  lastUpdated,
-  getCollectionId,
   collectionNameVersion,
+  getCollectionId,
+  lastUpdated,
+  tally,
 } from '../../utils/format';
-import Dropdown from '../DropDown/dropdown';
-import SimpleDropdown from '../DropDown/simple-dropdown';
-import Search from '../Search/search';
-import statusOptions from '../../utils/status';
 import pageSizeOptions from '../../utils/page-size';
-import List from '../Table/Table';
-import Bulk from '../Granules/bulk';
-import Overview from '../Overview/overview';
+import statusOptions from '../../utils/status';
+import isEqual from 'lodash.isequal';
 import {
-  tableColumns,
   reingestAction,
+  tableColumns,
 } from '../../utils/table-config/granules';
-import { strings } from '../locale';
-import DeleteCollection from '../DeleteCollection/DeleteCollection';
 import Breadcrumbs from '../Breadcrumbs/Breadcrumbs';
+import DeleteCollection from '../DeleteCollection/DeleteCollection';
+import Dropdown from '../DropDown/dropdown';
+import SimpleDropdown from '../DropDown/simple-dropdown';
+import Bulk from '../Granules/bulk';
 import ListFilters from '../ListActions/ListFilters';
+import { strings } from '../locale';
+import Overview from '../Overview/overview';
+import Search from '../Search/search';
+import List from '../Table/Table';
```

Comment:
These is automatic reordering of the imports, plus adding `isEqual` from lodash.